src/Components/TournamentDetails/helpers/__tests__/getListOfTournamentsFields.test.js

```diff
@@ -0,0 +1,62 @@
+
+import React from 'react';
+import {getListOfTournamentsFields} from "../getListOfTournamentsFields";
+import { renderToStaticMarkup } from 'react-dom/server';
+
+describe('getListOfTournamentsFields', () => {
+  it('выполнить return если tournament отсутствует(undefined)', () => {
+    const result = getListOfTournamentsFields(null);
+    expect(result).toBeUndefined();
+  });
+
+  it('Параметры турнира должны корректно отображаться', () => {
+    const tournament = {
+      place: "Великий Новгород",
+      country: "Россия",
+      covering: "Паркет",
+      format: "5x5",
+      games: 5,
+      teams: 24
+    };
+
+    const result = getListOfTournamentsFields(tournament);
+    const resultHtml = result.map(element => renderToStaticMarkup(element)).join('');
+
+    const expectedOutput = [
+      <span style={{ display: 'block' }} key={`value-1`}>город - <b>Великий Новгород</b></span>,
```

Comment:
(опционально)
Лучше выбрать какой-то один тип кавычек и его придерживать, лучше всего подходят одинарные кавычки для обычного текста и косые кавычки, если используешь переменные шаблонов вида \`value-${value}\`. Но это не ошибка, скорее придирка))
```suggestion
      <span style={{ display: 'block' }} key={'value-1'}>город - <b>Великий Новгород</b></span>,
```